sklearn/sklearn-template/template/trainer/metadata.py

```diff
@@ -19,17 +19,20 @@
 
 
 NUMERIC_FEATURES = [
```

Comment:
Everywhere: I talked more about 2 vs. 4 space indents in another PR comment, but gist is that for continuation lines, 4-space indent is the recommendation. We can discuss either doing 2-default/4-continue or 4/4-default/continue indent convention.

Reply:
Modified as 2-default/4-continue. Will change other model and input_util in the other PR to avoid conflict.